src/sound/sound_config.rs

```diff
@@ -20,6 +22,22 @@ impl SoundConfig {
         if let Some(bv) = sc.base_volume {
             self.base_volume = Some(bv);
         }
+        #[cfg(feature = "sound")]
+        // Load sounds configured in `[sound.collection]`.
+        // This doesn't "apply" this item in the sense other items are applied;
+        // but rather, add them to `super::play_sound::SOUNDS` so they can be
+        // looked up later.
+        // Ideally, they should be loaded at some more suitable point, or even
+        // passed to `super::play_sound::play_sound()` to achieve some degree
+        // of on-demand loading. But current structure of `SoundPlayer` makes
+        // that difficult.
+        if let Some(ref collection) = sc.collection {
+            for (name, path) in collection {
+                // TODO: Do we really want to panic here? Or should it be
+                // silenced?
+                crate::sound::play_sound::add_sound(name, path).unwrap();
```

Comment:
Why would we want to have a panic ? That's not how bacon should behave

Reply:
Indeed, hence the TODO.

Do we silence it, or warn about it? If warn, where should the warning go?

Reply:
Indeed, hence the TODO.

Do we silence it, or warn about it? If warn, where should the warning go?